charts/internal/calico/templates/node-cpva/configmap-node-vertical-autoscaling.yaml

```diff
@@ -0,0 +1,20 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: calico-node-vertical-autoscaler
+  namespace: kube-system
+data:
+  node-autoscaler: |-
+    {
+      "calico-node": {
+        "requests": {
+          "cpu": {
+            "base": "120m",
+            "step": "80m",
+            "nodesPerStep": 10,
+            "max": "1000m"
+          }
+        }
+      }
+    }
```

Comment:
Does this config mean that it won't scale the limits proportionally (is it even capable of doing so)? I.e., if the limits are reached, still nothing happens?

Reply:
> Does this config mean that it won't scale the limits proportionally
It does scale proportionally based on the `nodes-per-step`

> is it even capable of doing so? 
why not? it watches nodes and updates daemonsets

> if the limits are reached, still nothing happens?
limits are updated.